### ISSUE TITLE
Speed prediction training

### ISSUE THREAD

**Issue by ak359** [mayankmalik01/Self-driving-car#1]
Hi , congratulations for great work!
I was trying to replicate speed prediction task for custom dataset but my my  validation loss values does not decrease below 40!
Trained for 1000 epochs with all default training parameters.
Any suggestions??

Thank you
